include/albatross/src/covariance_functions/radial.hpp

```diff
@@ -59,6 +59,28 @@ class SquaredExponential
     return "squared_exponential[" + this->distance_metric_.get_name() + "]";
   }
 
+  std::vector<double> _ssr_impl(const std::vector<double> &xs) const {
+    double min = *std::min_element(xs.begin(), xs.end());
+    double max = *std::max_element(xs.begin(), xs.end());
+
+    double range = max - min;
+
+    // using 1/10th of the length scale should result in grids with
+    // one percent decorrelation between them.
```

Comment:
Wouldn't users want different granularity depending on application?

Reply:
Yeah good chance, I do think this is a relatively disciplined approach though and if users want something different they can use their own `InducingPointStrategy`